src/main/java/gregtech/common/CommonProxy.java

```diff
@@ -173,6 +174,7 @@ public static void registerRecipesLowest(RegistryEvent.Register<IRecipe> event)
         DecompositionRecipeHandler.runRecipeGeneration();
         RecyclingRecipes.init();
         WoodMachineRecipes.init();
+        MetaItemBracketHandler.rebuildComponentRegistry();
```

Comment:
Based on what did you choose this place?

Reply:
This is the same registry spot that CraftTweaker uses for their bracket handlers.

Reply:
That seems like good place then.